galley/docker/Dockerfile.galley

```diff
@@ -11,7 +12,7 @@ USER 1337:1337
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
 # hadolint ignore=DL3007
-FROM gcr.io/distroless/static:nonroot as distroless
+FROM ${DISTROLESS_IMAGE}/static:nonroot as distroless
```

Comment:
I could see how this could work. This is a better approach than that used in Dockerfile.istiod above.